pkg/machinepool/machinepool.go

```diff
@@ -955,11 +955,13 @@ func (m *machinePool) AddNodePool(cmd *cobra.Command, clusterKey string, cluster
 			return fmt.Errorf("Expected a valid http tokens value : %v", err)
 		}
 	}
-	if err = ocm.ValidateHttpTokensValue(httpTokens); err != nil {
-		return fmt.Errorf("Expected a valid http tokens value : %v", err)
-	}
-	if err := ocm.ValidateHttpTokensVersion(ocm.GetVersionMinor(version), httpTokens); err != nil {
-		return fmt.Errorf(err.Error())
+	if httpTokens != string(cmv1.Ec2MetadataHttpTokensOptional) && version != "" {
```

Comment:
the panic error only happened in ValidateHttpTokensVersion func when version is nil or ""?
if that's true, I think the simplest solution is to remove the func here since it's outdated. 
It check cluster version no less than 4.11, which is old and not supported now.

Reply:
Yeah it panics when the version is "", I'll remove that check in that case